.circleci/config.yaml

```diff
@@ -0,0 +1,16 @@
+version: 2.1
+orbs:
+  architect: giantswarm/architect@0.1.0
+
+workflows:
+  package-and-push-chart-on-tag:
+    jobs:
+      - architect/push-to-app-catalog:
+          name: "package and push elastic-logging chart"
+          app_catalog: "giantswarm-incubator-catalog"
+          app_catalog_test: "giantswarm-incubator-catalog"
```

Comment:
Should these be the same? Or should we also create a test catalog?

Reply:
Yes, I'm creating test one (and will be updating config accordingly), as agreed with @puja108 last night https://gigantic.slack.com/archives/C76JX6YLQ/p1560277309187500?thread_ts=1560255130.171900&cid=C76JX6YLQ

Reply:
Technically this is ok to push both to the same catalog. But I guess you wan to push branch artifacts to "giantswarm-incubator-test-catalog"

Reply:
Thanks. Yes good to create a test catalog since the incubator is customer facing.

Reply:
For both GA/promoted and for incubating catalogs, test catalog variants have been created and Circle CI configuration adjusted.
So, currently there's:
- giantswarm-catalog
- giantswarm-test-catalog
- giantswarm-incubator-catalog
- giantswarm-incubator-test-catalog

Currently we use just incubator + incubator-test catalog. GA promotion conditions and process are to be defined.